test/cli/metrics-file/metrics-file.sh

```diff
@@ -27,6 +27,7 @@ grep -A1 "\"ruby_typer.unknown..types.input.files.sigil.false\"" metrics4.json
 grep -A1 "\"ruby_typer.unknown..types.input.files.sigil.true\"" metrics4.json
 grep -A1 "\"ruby_typer.unknown..types.input.modules.total\"" metrics4.json
 grep -A1 "\"ruby_typer.unknown..types.input.classes.total\"" metrics4.json
+grep -A1 "\"ruby_typer.unknown..singleton_classes\"" metrics4.json
```

Comment:
It was already displayed as we test with a debug build, I just wanted to make it more explicit as it's now a first-class metrics.